dataland-backend/src/main/kotlin/org/dataland/datalandbackend/model/p2p/categories/commercialRealEstate/P2pCommercialRealEstate.kt

```diff
@@ -0,0 +1,17 @@
+package org.dataland.datalandbackend.model.p2p.categories.commercialRealEstate
+
+import org.dataland.datalandbackend.model.p2p.categories.commercialRealEstate.subcategories.P2pCommercialRealEstateBuildingEfficiency
+import org.dataland.datalandbackend.model.p2p.categories.commercialRealEstate.subcategories.P2pCommercialRealEstateEnergySource
+import org.dataland.datalandbackend.model.p2p.categories.commercialRealEstate.subcategories.P2pCommercialRealEstateTechnology
+
+/**
+ * --- API model ---
+ * Fields of the category "Commercial Real Estate" of the p2p framework.
+*/
+data class P2pCommercialRealEstate(
+    val buildingEfficiency: P2pCommercialRealEstateBuildingEfficiency? = null,
```

Comment:
As stated before should be unified to one real estate setup that is reused for commercial and residential.

Reply:
Reverted, there is now only the RealEstate Class again